### ISSUE TITLE
[i18n][feature] use numeric value as discrete cases for NgPlural

### ISSUE THREAD

**Issue by vicb** [angular/angular#11578]
Current usage

``` html
<some-element [ngPlural]="value">
  <ng-container *ngPluralCase="'=0'">there is nothing</ng-container>
  <ng-container *ngPluralCase="'=1'">there is one</ng-container>
  <ng-container *ngPluralCase="'few'">there are a few</ng-container>
  <ng-container *ngPluralCase="'other'">there are exactly #</ng-container>
</some-element>
```

Also allow numeric values (ie 0, 1 below):

``` html
<some-element [ngPlural]="value">
  <ng-container *ngPluralCase="0">there is nothing</ng-container>
  <ng-container *ngPluralCase="1">there is one</ng-container>
  <ng-container *ngPluralCase="'few'">there are a few</ng-container>
  <ng-container *ngPluralCase="'other'">there are exactly #</ng-container>
</some-element>
```

**Comment by cexbrayat** [comment]
@vicb this can be closed now that @mlaval 's PR #13876 has been merged, right?

**Comment by mlaval** [comment]
Sure
